flatson/flatson.py

```diff
@@ -71,7 +55,10 @@ class Flatson(object):
         'join_values': join_values,
     }
 
-    def __init__(self, schema, field_sep='.'):
+    def __init__(self, schema, field_sep='.', ignore=None):
+        self.ignore = ignore
+        if not self.ignore:
+            self.ignore = []
```

Comment:
Let's make the ignore argument handling:

```
self.ignore = set(ignore) if ignore else set()
```

It's nice to use a set here because we're only using it for containment check. ;)